### PR TITLE
Remove prometheus rebase

### PR DESCRIPTION
**Pull request by mayerm94**
Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR. https://oncprojectracking.healthit.gov/support/browse/LANTERN-220
  - The JIRA ticket for this PR is: 
- [x] Tests are complete. NA
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

Create a new branch `<branch name>_gomodupdate`. Run `make update_mods` to update the `go.mod`
and `go.sum` files to point to master. Create a PR. Require one sanity check reviewer.

### CODE REVIEW

File: endpointmanager/README.md [hunk 1]
```diff
@@ -192,4 +184,23 @@ To run, perform the following commands:
 ```bash
 cd endpointmanager/cmd/nppesorgpopulator
 go run main.go <path to nppes csv file>
-```
\ No newline at end of file
+```
+
+### Expected Endpoint Source Formatting
+
+The Network Statistics Querier expects the format of an endpoint source list to be in the below format, unless one of the exceptions noted below.
```
Comment (emichaud998) [inline endpointmanager/README.md:191]: Not sure if this should say Network Statistics Querier here since the Network Stats Querier was removed

**Comment by emichaud998** [comment]
In the definitions.json file I noticed that in the queues list there is an entry with the name "endpoints-to-netstats", which I assume might be able to be removed

File: docker-compose.yml [hunk 2]
```diff
@@ -1,25 +1,8 @@
...
-        - LANTERN_QPASSWORD=${LANTERN_QPASSWORD}
-        - LANTERN_QHOST=${LANTERN_QHOST}
-        - LANTERN_QPORT=${LANTERN_QPORT}
-        - LANTERN_ENDPTINFO_NETSTATS_QNAME=${LANTERN_ENDPTINFO_NETSTATS_QNAME}
-        - LANTERN_ENDPTQRY_NUMWORKERS=${LANTERN_ENDPTQRY_NUMWORKERS}
-    volumes:
-        - ./scripts/wait-for-it.sh:/etc/lantern/wait-for-it.sh
-    command: /etc/lantern/wait-for-it.sh lantern-mq:5672 -- ./main
-
   pg_prometheus:
```
Comment (karenzheung) [inline docker-compose.yml:4]: docker image name should be changed

Comment (karenzheung) [reply docker-compose.yml:4]: There are other instances where lantern_dbhost = pg_prometheus that will need to be changed as well

**Pull request merged**